### DISCUSSION TITLE
How to apply a BaseComponent to all components without explicitly adding "@inherits" directive

### DISCUSSION THREAD

**Discussion by bancroftway** [dotnet/aspnetcore#57048] [Q&A]
I recall that a few years ago, there was a way in Blazor Wasm project to auto-inherit from a base class for all the components in the project, instead of having to add "@inherits" directive to all components and pages. Has this feature been removed, as I cannot find any discussion or documentation on this? Is this still possible to do? Thanks

**Comment by KennethHoff** [comment]
What happens if you add the directive to the `_Imports.razor` file in the root directory?
